VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -241,4 +257,26 @@ public void onActivityStarted(Activity activity) {
         public void onActivityStopped(Activity activity) {
         }
     }
+
+    public void clearWebViewCache(){
+        try {
+            boolean success = deleteDatabase("webview.db");
```

Comment:
@aleffert i did some research, looks it is the approach most people follows.

WebView has a CacheManager class, but it is deprecated and no longer available.  i read the CacheManager source code, looks android stores the http request history in webview.db.